examples/hello_rtneural/Makefile

```diff
@@ -1,4 +1,4 @@
-CXXFLAGS = -O3 -I../../ -std=c++17 -DRTNEURAL_DEFAULT_ALIGNMENT=16
+CXXFLAGS = -O3 -I../../ -std=c++17 -DRTNEURAL_DEFAULT_ALIGNMENT=16 -DRTNEURAL_NAMESPACE=RTNeural
```

Comment:
Ah, I hadn't thought about this... maybe instead it would be better to have something at the top of "RTNeural.h" like:
```cpp
#ifndef RTNEURAL_NAMESPACE
#define RTNEURAL_NAMESPACE RTNeural
#endif
```

That way folks currently using RTNeural without CMake won't have a breaking change.

Reply:
smart. I've pushed a new commit.